aiven/client/cli.py

```diff
@@ -40,6 +40,11 @@
     "create_time",
     "update_time",
 ]
+USER_GROUP_COLUMNS = [
+    "user_group_name",
+    "description",
+    "user_group_id",
```

Comment:
I feel somewhat biased to having the "primary key" be the leftmost column, as it's more easily findable/selectable by a human who'll likely copy paste it for the next command where they need to specify that value ... and second column be the friendly human readable name, cause ideally it would be close by for easy matching up?


```suggestion
    "user_group_id",
    "user_group_name",
    "description",
```

Reply:
Made `user_group_id` the second.